tests/common/devices.py

```diff
@@ -327,3 +327,37 @@ def __init__(self, ansible_adhoc, hostname, user, passwd, gather_facts=False):
                   'ansible_password': passwd, \
                   'ansible_become_method': 'enable' }
         self.host.options['variable_manager'].extra_vars.update(evars)
+
+    def shutdown(self, interface_name):
```

Comment:
TODO
This interfaces should implement in SONiCHost too.
I will use host.command() to implement it